scripts/download_srs_aztec.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ -f "$AZTEC_SRS_PATH" ]; then
+    echo "SRS file $AZTEC_SRS_PATH exists"
+else
+    echo "SRS file $AZTEC_SRS_PATH does not exist, downloading ..."
+    gh release download --repo alxiong/ark-srs v0.2.0 -p "$(basename $AZTEC_SRS_PATH)" -O "$AZTEC_SRS_PATH" --skip-existing
```

Comment:
Why not download with wget? E. g.

```
wget https://github.com/alxiong/ark-srs/releases/download/v0.2.0/kzg10-aztec20-srs-1048584.bin
```

Reply:
I tried with curl -L and wget and many other combinations of option flags, but while running them works directly in terminal, running them in bash script doesn't.

And the gh CLI is the simplest solution i can find.

Reply:
Weird. It seems to work for me to run this wget command in bash script. It's true curl seems to be troublesome though.

Reply:
Oh that's interesting, can you double check the sha256sum of the downloaded against https://github.com/alxiong/ark-srs/blob/main/src/constants.rs#L11?

If wget works for you, then I'll try again tmr.

Reply:
you are right, wget can be done, [63c40c3](https://github.com/EspressoSystems/espresso-sequencer/pull/1329/commits/63c40c3069ac3a2d00aa435e0e5b36466e79fc15)
(previously I was testing locally outside the nix-shell, not sure it wasn't work then, nvm)